waimak-rdbm-ingestion/src/test/scala/com/coxautodata/waimak/rdbm/ingestion/SQLServerExtractorIntegrationTest.scala

```diff
@@ -34,54 +34,42 @@ class SQLServerExtractorIntegrationTest extends SparkAndTmpDirSpec with BeforeAn
-        |        insert into TestTemporal (TestTemporalID, TestTemporalValue) VALUES (3, 'Value3');
-        |        insert into TestTemporal (TestTemporalID, TestTemporalValue) VALUES (4, 'Value4');
-        |        insert into TestTemporal (TestTemporalID, TestTemporalValue) VALUES (5, 'Value5');
-        |        insert into TestTemporal (TestTemporalID, TestTemporalValue) VALUES (6, 'Value6');
-        |        insert into TestTemporal (TestTemporalID, TestTemporalValue) VALUES (7, 'Value7');
-        |        update TestTemporal set TestTemporalValue = 'New Value 1' where TestTemporalID = 1""".stripMargin
 
-    val testNonTemporalTableCreate =
-      """CREATE TABLE TestNonTemporal
-        |(
-        |     TestNonTemporalID1 int NOT NULL
-        |   , TestNonTemporalID2 int NOT NULL
-        |   , TestNonTemporalValue varchar(50) NOT NULL
-        |   PRIMARY KEY CLUSTERED(TestNonTemporalID1, TestNonTemporalID2)
-        |)
-        |;
-        |
-        |   insert into TestNonTemporal (TestNonTemporalID1, TestNonTemporalID2, TestNonTemporalValue) VALUES (1, 1, 'V1');
-        |   insert into TestNonTemporal (TestNonTemporalID1, TestNonTemporalID2, TestNonTemporalValue) VALUES (2, 1, 'V2');
-        |   insert into TestNonTemporal (TestNonTemporalID1, TestNonTemporalID2, TestNonTemporalValue) VALUES (2, 2, 'V3');
-        |   insert into TestNonTemporal (TestNonTemporalID1, TestNonTemporalID2, TestNonTemporalValue) VALUES (4, 3, 'V4');
-        |   insert into TestNonTemporal (TestNonTemporalID1, TestNonTemporalID2, TestNonTemporalValue) VALUES (5, 3, 'V5');""".stripMargin
+    val testTableCreate =
+      s"""CREATE TABLE testtable
+         |(
+         |     testtableID1 int NOT NULL
+         |   , testtableID2 int NOT NULL
+         |   , testtableValue varchar(50) NOT NULL
+         |   PRIMARY KEY CLUSTERED(testtableID1, testtableID2)
+         |)
+         |;
+         |
+        |   insert into testtable (testtableID1, testtableID2, testtableValue) VALUES (1, 1, 'V1');
+         |   insert into testtable (testtableID1, testtableID2, testtableValue) VALUES (2, 1, 'V2');
+         |   insert into testtable (testtableID1, testtableID2, testtableValue) VALUES (2, 2, 'V3');
+         |   insert into testtable (testtableID1, testtableID2, testtableValue) VALUES (4, 3, 'V4');
+         |   insert into testtable (testtableID1, testtableID2, testtableValue) VALUES (5, 3, 'V5');""".stripMargin
+
+    val testTableSuffixCreate =
+      s"""
+         |CREATE TABLE testtable_suffix(
+         |   testtableID1 int NOT NULL
+         | , testtableID2 int NOT NULL
+         | , testtableValue varchar(50) NOT NULL
+         | , testtable_last_updated datetime2 NOT NULL
+         |   PRIMARY KEY CLUSTERED(testtableID1, testtableID2)
+         |);
+         |        insert into testtable_suffix (testtableID1, testtableID2, testtableValue) VALUES (6, 4, 'V6', '$insertTimestamp');
```

Comment:
Can you add a blank line and make the indentation for the insert the same as `testTableCreate` so it looks consistent.